src/main/java/org/kiwiproject/test/dropwizard/app/PostgresAppTestExtension.java

```diff
@@ -0,0 +1,63 @@
+package org.kiwiproject.test.dropwizard.app;
+
+import io.dropwizard.Application;
+import io.dropwizard.Configuration;
+import io.dropwizard.testing.ConfigOverride;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.zonky.test.db.postgres.embedded.LiquibasePreparer;
+import io.zonky.test.db.postgres.junit5.EmbeddedPostgresExtension;
+import io.zonky.test.db.postgres.junit5.PreparedDbExtension;
+import lombok.Getter;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * Multi-purpose extension for {@link Application} testing requiring a postgres database. The extension will spin up an
```

Comment:
Probably ought to mention that we are assuming Liqiubase migrations here. That embedded Postgres actually supports both Liquibase and Flyway. Don't think we need to support both here, but should maybe mention it.

Reply:
Done